sotodlib/site_pipeline/preprocess_tod.py

```diff
@@ -170,3 +197,40 @@ def load_preprocess_tod(obs_id, configs="preprocess_configs.yaml", context=None
         logger.info(f"Processing {process.name}")
         process.process(aman, aman.preprocess)
     return aman
+
+
+def get_parser(parser=None):
+    if parser is None:
+        parser = argparse.ArgumentParser()
+    parser.add_argument('configs', help="Preprocessing Configuration File")
+    parser.add_argument(
+        '--query', 
+        help="Query to pass to the observation list", 
+        default=2, 
```

Comment:
Heh, worried that query(1) won't quite get everything?  Is None not equivalent to 1?  That would be a bug in my books...

Reply:
yeah fair, I put in the query because I was doing something with a database that needed to exclude entries based on a calibration column. But between this and your other suggestions I'll expand the options / flags a but more.